dask_cuda/cli.py

```diff
@@ -164,6 +164,17 @@ def cuda():
         incompatible with RMM pools and managed memory, trying to enable both will
         result in failure.""",
 )
+@click.option(
+    "--set-rmm-allocator-for-libs",
+    "rmm_allocator_external_lib_list",
+    type=CommaSeparatedChoice(["cupy", "torch"]),
+    default=None,
+    show_default=True,
+    help="""
+    Set RMM as the allocator for external libraries. Provide a comma-separated
+    list of libraries to set, e.g., "torch,cupy".
+    Supported options are: torch, cupy.""",
```

Comment:
```suggestion
    list of libraries to set, e.g., "torch,cupy".""",
```

Reply:
This can be removed in favor of the `CommaSeparatedChoice`-generated help, this is what it looks like after applying the change above:

```
  --set-rmm-allocator-for-libs [cupy|torch]
                                  Set RMM as the allocator for external libraries.
                                  Provide a comma-separated list of libraries to set,
                                  e.g., "torch,cupy".
```

Reply:
Resolved by: https://github.com/rapidsai/dask-cuda/pull/1392/commits/33871acd1ace384e1237f2caaf4a041a2c1b7cbf